bulletin-of-the-seismological-society-of-america.csl

```diff
@@ -0,0 +1,186 @@
+      <label form="short" prefix=" (" suffix=")"/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <text variable="title"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" and="text" delimiter=", " initialize-with="."/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <choose>
+          <if type="bill book graphic legal_case legislation motion_picture report song" match="any">
+            <text variable="title" form="short" font-style="italic"/>
+          </if>
+          <else>
+            <text variable="title" form="short" quotes="true"/>
+          </else>
+        </choose>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="publisher">
+    <choose>
+      <if type="article-journal article-magazine" match="any">
+        <group delimiter=", ">
+          <text variable="container-title" form="short"/>
+          <text variable="volume"/>
+          <group delimiter=" ">
+            <label variable="issue" form="short"/>
+            <text variable="issue"/>
+          </group>
+        </group>
+      </if>
+      <else>
+        <group delimiter=", ">
+          <text variable="publisher"/>
+          <choose>
+            <if type="paper-conference" match="none">
+              <text variable="publisher-place"/>
+            </if>
+          </choose>
+          <group delimiter=" ">
+            <text variable="collection-title"/>
+            <text variable="collection-number"/>
+          </group>
+        </group>
+      </else>
+    </choose>
+  </macro>
+  <macro name="year-date">
+    <choose>
+      <if variable="issued">
+        <date variable="issued">
+          <date-part name="year"/>
+        </date>
+      </if>
+      <else>
+        <text term="no date" form="short"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="book-edited">
+    <choose>
+      <if type="chapter paper-conference" match="any">
+        <group delimiter=" ">
+          <text term="in" font-style="normal" suffix=" "/>
+          <text variable="container-title" font-style="italic"/>
+          <names variable="editor translator" prefix=" " delimiter=", ">
+            <name and="text" sort-separator=", " initialize-with=". " delimiter-precedes-last="always"/>
+            <label text-case="capitalize-first" prefix=" (" suffix=")"/>
+          </names>
+          <choose>
+            <if type="paper-conference">
+              <text variable="event-place"/>
+            </if>
+          </choose>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="mapinfo">
+    <choose>
+      <if type="map">
+        <text variable="note"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="accessed">
+    <choose>
+      <if type="webpage post-weblog">
```

Comment:
needs `match="any"` (otherwise defaults to `match="all"` even though Zotero/citeproc-js may not implement that correctly.

Reply:
I wonder how that would've happened. I did this in the visual editor like always (click on plus and add post-weblog). thanks for spotting.